<?xml version="1.0"?>
<launch>
  <arg name="robotname" default="sos1"/>
  <group ns="$(arg robotname)">
  <node   pkg="gmapping"   type="slam_gmapping" name="mapping">
    <param name="map_frame" value="/map" />
    <param name="base_frame" value="$(arg robotname)/base_link"/>
    <param name="odom_frame" value="$(arg robotname)/odom" />
    <remap from="scan" to="/$(arg robotname)/hokuyo" />
    <param name="map_update_interval" value="0.1" />
    <param name="delta" value="0.2 "/>
    <param name="particle" value="5" />
    <param name="iterations" value="1" />
    <param name="maxUrange" value="29" />
    <param name="maxRange" value="30.4" />
    <param name="linearUpdate" value="0.1" />
    <param name="xmin" value="-125" />
    <param name="xmax" value="125" />
    <param name="ymax" value="125" />
    <param name="ymin" value="-125" />

  </node>
</group>
</launch>
